src/ProduceCode.lhs

```diff
@@ -1319,7 +1320,7 @@ slot is free or not.
 > hexChars acts = concat (map hexChar acts)
 
 > hexChar :: Int -> String
-> hexChar i | i < 0 = hexChar (i + 2^16)
+> hexChar i | i < 0 = hexChar (i + 65536)
```

Comment:
It prevents a compiler warning (type defaulting due to `^`). Most software engineers should recognise `65536` as `1 << 17` or `2^16`.